products/views.py

```diff
@@ -22,4 +22,25 @@ def get(self,request):
                 } for subcategory in category.sub_categories.all()]
             } for category in categories]
 
-        return JsonResponse({'message':'SUCCESS' ,'Category' : result}, status=200)
\ No newline at end of file
+        return JsonResponse({'message':'SUCCESS' ,'Category' : result}, status=200)
+
+class MainPageProductList(View):
```

Comment:
페이지 기준으로 엔드포인트를 만드시면 안됩니다.
이런 방식으로 엔드포인트를 구현하게 되면, 매번 client에서 새로운 페이지를 만들때마다 엔드포인트가 끊임없이 늘어납니다.

엔드포인트 Restful하게 즉 자원(data) 중심으로 만들어주셔야 합니다. View class명도 `ProductList`(상품 목록)으로 수정해주세요.

```suggestion
class ProductList(View):
```